src/traces/scatter/attributes.js

```diff
@@ -65,6 +65,10 @@ module.exports = {
             'See `y0` for more info.'
         ].join(' ')
     },
+    key: {
```

Comment:
@rreusser how are these different to `trace.uid` ?

Is there a case, in general, where multiple traces would share the same `key` ?